src/Tizen.NUI.Components/Controls/SelectButton.cs

```diff
@@ -198,17 +220,31 @@ protected virtual void OnSelectedChanged()
         private void Initialize()
         {
             IsSelectable = true;
-        }
 
-        private void OnSelect()
-        {    
-            OnSelectedChanged();
+            wasSelected = IsSelected;
+
+            PropertyChanged += SelectButtonPropertyChanged;
```

Comment:
In this case I think it'd better to override `OnControlStateChanged`, instead listening `PropertyChanged`.
```C#
protected override void OnControlStateChanged(ControlStateChangedEventArgs info)
{
    if (info.PreviousState.Contains(ControlState.Selected) != info.CurrentState.Contains(ControlState.Selected))
    {
        // The selection has changed
    }
}
```